parser/cpp/AstNode.h

```diff
@@ -20,10 +20,36 @@ class AstNode : public SimpleNode {
   Token* endToken;
   virtual ~AstNode() {}
   AstNode(int id) : SimpleNode(id) {}
+  
   AstNode(SqlParser* parser, int id) : SimpleNode(parser, id) {}
-  int NumChildren() { return jjtGetNumChildren(); }
-  AstNode* GetChild(int i) { return static_cast<AstNode*>(jjtGetChild(i)); }
-  JJString toString(const JJString& prefix) const {
+
+  int Kind() const { return id; }
```

Comment:
Should function name start with lower case?

Reply:
Yeah - it's one of the weird things that happened when we did C++ first and then java. Will fix it.

Reply:
Actually, in C++ the style is upper case starting letter. So not changing it here.